slaves/linux-cross/build_arm_toolchain.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Perhaps this script could be called just `build_toolchain.sh`?  Looks like we could funnel a bunch of other toolchains through here as well.